packages/inference/README.md

```diff
@@ -117,7 +117,7 @@ for await (const output of hf.textGenerationStream({
 
 ### Text Generation (Chat Completion API Compatible)
 
-Using the `chatCompletion` method, you can generate text with models compatible with the OpenAI Chat Completion API. All models served by [TGI](https://api-inference.huggingface.co/framework/text-generation-inference) on Hugging Face support Messages API.
```

Comment:
kinda killed that URL (the /framework one), prob not really relevant anymore anyways cc @XciD @Wauplin

Reply:
let's keep it for a bit at least for backward compatibility :pray: (but yeah, no need for it anymore)

`huggingface_hub` has a method relying on it